site/assets/scss/_content.scss

```diff
@@ -104,3 +112,31 @@
     border-left: var(--#{$prefix}border-width) solid $border-color;
```

Comment:
```suggestion
    border-left: var(--bs-border-width) solid var(--bs-border-color);
```

Reply:
Yes + check in this file other occurences

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/baf49030e10fc0d2d0c49d235323ad5e2bdb53ed